packages/wp-story-editor/src/api/pageTemplate.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { addQueryArgs } from '@web-stories-wp/design-system';
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+export function getCustomPageTemplates(page = 1, customPageTemplates) {
+  let apiPath = customPageTemplates;
+  const perPage = 100;
+  apiPath = addQueryArgs(apiPath, {
+    context: 'edit',
+    per_page: perPage,
+    page,
+    _web_stories_envelope: true,
+  });
```

Comment:
```suggestion
  const perPage = 100;
  const apiPath = addQueryArgs(customPageTemplates, {
    context: 'edit',
    per_page: perPage,
    page,
    _web_stories_envelope: true,
  });
```

Reply:
Done in https://github.com/google/web-stories-wp/pull/8966/commits/a332ebafd61dfc4a7f9ceddb84412e739ce7b117